features/bevragen/persoon/adressering/adres-regels/dev/fields-gba.feature

```diff
@@ -0,0 +1,432 @@
+
+  Abstract Scenario: persoon heeft een verblijfplaats buitenland
+    Gegeven de persoon met burgerservicenummer '000000346' heeft de volgende 'verblijfplaats' gegevens
+    | land (13.10) | regel 1 adres buitenland (13.30) | regel 2 adres buitenland (13.40) | regel 3 adres buitenland (13.50) |
+    | 5010         | Rue du pomme 26                  | Bruxelles                        | postcode 1000                    |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000346                       |
+    | fields              | adressering.<veld>              |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+    | naam              | waarde          |
+    | land.code         | 5010            |
+    | land.omschrijving | België          |
+    | regel1            | Rue du pomme 26 |
+    | regel2            | Bruxelles       |
+    | regel3            | postcode 1000   |
+
+    Voorbeelden:
+    | veld        |
+    | adresregel1 |
+    | adresregel2 |
+    | adresregel3 |
+    | land        |
+
+  Abstract Scenario: persoon heeft een verblijfplaats buitenland en 'land (13.10)' heeft de standaardwaarde
+    Gegeven de persoon met burgerservicenummer '000000346' heeft de volgende 'verblijfplaats' gegevens
+    | land (13.10) | regel 1 adres buitenland (13.30) | regel 2 adres buitenland (13.40) | regel 3 adres buitenland (13.50) |
+    | 0000         | Rue du pomme 26                  | Bruxelles                        | postcode 1000                    |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000346                       |
+    | fields              | adressering.<veld>              |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+    | naam              | waarde          |
+    | land.code         | 0000            |
+    | land.omschrijving | Onbekend        |
+    | regel1            | Rue du pomme 26 |
+    | regel2            | Bruxelles       |
+    | regel3            | postcode 1000   |
+
+    Voorbeelden:
+    | veld        |
+    | adresregel1 |
+    | adresregel2 |
+    | adresregel3 |
+    | land        |
+
+  Scenario: persoon heeft een verblijfplaats buitenland en 'regel 1 adres buitenland (13.30)', 'regel 2 adres buitenland (13.40)' en 'regel 3 adres buitenland (13.50)' hebben de standaardwaarde
+    Gegeven de persoon met burgerservicenummer '000000346' heeft de volgende 'verblijfplaats' gegevens
+    | land (13.10) | regel 1 adres buitenland (13.30) | regel 2 adres buitenland (13.40) | regel 3 adres buitenland (13.50) |
```

Comment:
waarom hier met "standaardwaarden" (lege string?) voor de regels? En niet afwezig zijn van die waarden (alleen opgeven land) zoals we normaal doen? Is dat iets anders?